query/influxql/compiler.go

```diff
@@ -62,7 +62,7 @@ func (c *Compiler) Compile(ctx context.Context) (flux.Program, error) {
 		return nil, err
 	}
 	compileOptions := lang.WithLogPlanOpts(c.logicalPlannerOptions...)
-	return lang.CompileAST(astPkg, now, compileOptions), nil
+	return lang.CompileAST(astPkg, now, compileOptions)
```

Comment:
I'm not sure I understand this change.  We are on flux v0.37.2 in influxdb now, and this branch is on 0.32.1... what am I missing?

Reply:
This is a super old PR that wasn't merged in flux and got closed. Closing this one too.